tool/truffle/jruby_truffle_runner/gem_configurations/activesupport.yaml

```diff
@@ -0,0 +1,24 @@
+---
+:global:
+  :jruby_truffle_path: '../../jruby/bin/ruby'
+:setup:
+  :after:
+    - 'echo "$:.unshift \"\#{path}/..\"" >> "#{options[:global][:truffle_bundle_path]}/bundler/setup.rb"'
+  :file:
+    shims.rb: |
+              module Minitest
+                module Assertions
+                  def self.diff
+                    nil
+                  end
+                end
+              end
+    bundler.rb: "module Bundler; def self.setup; end; end"
+    method_source.rb: nil
```

Comment:
The nil is interpreted as a String which is intended. I needed to replace `method_source` gem with empty file. We have some bug though when parsing empty file hence the nil. I'll add comments to the yaml file.